packages/cyberstorm/src/components/PackageFlag.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageFlag.module.css";
+
+export interface PackageFlagProps {
+  label?: string;
+  icon?: ReactNode;
+  packageFlagStyle?: string;
+}
+
+/**
+ * Cyberstorm PackageFlag component
+ */
+export const PackageFlag: React.FC<PackageFlagProps> = (props) => {
+  const { label, icon } = props;
+  styles;
+  return (
+    <button type="button" className={"packageFlag"}>
+      {icon ? <div className="icon__rotate_30">{icon}</div> : null}
+      {label ? <div className="packageFlagLabel">{label}</div> : null}
```

Comment:
Also an agreed on naming convention, the class name here should be just `label`, since it's obvious from the context that it's for the package flag.